gamification/src/main/java/juja/microservices/gamification/user/UserRepositoryCustom.java

```diff
@@ -29,10 +29,9 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-package juja.microservices.gamification.model.repository;
+package juja.microservices.gamification.user;
```

Comment:
@sLisnychyi remove pls `UserRepositoryCustom`. I suppose - it is not required any more